gh-77521: Add link to builtin module names (#92438)
python/cpython: Doc/tutorial/modules.rst

.. index:: triple: module; search; path

When a module named :mod:`spam` is imported, the interpreter first searches for

> a built-in module with that name. These module names can be found from

Reviewer: ```suggestion
a built-in module with that name. These module names are listed in
```
Shorter wording
